lib/generators/mi/mi_generator.rb

```diff
@@ -21,15 +21,24 @@ def self.next_migration_number(dirname)
   source_root File.expand_path('../templates', __FILE__)
 
   def doing
+    if arguments.include?('--version')
+      puts Mi::VERSION
```

Comment:
__[rubocop]__
* Rails/Output: Do not write to stdout. Use Rails' logger if you want to log.